docs/meshstack.aws.metering.md

```diff
@@ -0,0 +1,22 @@
+---
+id: meshstack.aws.metering
+title: Metering
+---
+
+meshStack imports metering data from `AWS Cost and Usage Reports`
+
+The report should have the following properties
```

Comment:
Please make it clearer that they have to configure the data in the report. Maybe attach a link to AWS where it is explained how to do it.

Reply:
Updated with links to the AWS docs